Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -299,6 +299,8 @@ public void StartClient(string serverIp)
 
             isNetworkActive = true;
 
+            client.Authenticated.AddListener(OnAuthenticated);
```

Comment:
what happens if I startClient,  then stop it and then start again?  woudn't this register twice?

perhaps this can be done on Start()?

Reply:
fixed